examples/midwest_survey.py

```diff
@@ -20,19 +21,19 @@
 from sklearn.ensemble import RandomForestClassifier
 from sklearn.model_selection import StratifiedKFold
 
-from dirty_cat import datasets
+import dirty_cat.datasets.fetching as fetching
```

Comment:
We should import directly from datasets

Reply:
not sure I see how to do it, since examples is in the root folder, (dirty_cat) whereas 
datasets is in (dirtycat/dirtycat)